x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/agent_config/components/config_form.tsx

```diff
@@ -221,16 +221,20 @@ export const AgentConfigForm: React.FunctionComponent<Props> = ({
           options={[
             {
               id: 'logs',
-              label: i18n.translate(
-                'xpack.ingestManager.agentConfigForm.monitoringLogsFieldLabel',
-                { defaultMessage: 'Collect agent logs' }
+              label: (
+                <FormattedMessage
+                  id="xpack.ingestManager.agentConfigForm.monitoringLogsFieldLabel"
+                  defaultMessage="Collect agent logs"
+                />
               ),
             },
             {
               id: 'metrics',
-              label: i18n.translate(
-                'xpack.ingestManager.agentConfigForm.monitoringMetricsFieldLabel',
-                { defaultMessage: 'Collect agent metrics' }
+              label: (
+                <FormattedMessage
+                  id="xpack.ingestManager.agentConfigForm.monitoringMetricsFieldLabel"
+                  defaultMessage="Collect agent metrics"
+                />
```

Comment:
@hbharding we discussed adding tooltips to these checkbox labels, but I couldn't think of any content that wouldn't be redundant to what these labels already say. happy to add tooltips if you or @dedemorton have any suggestions

Reply:
It might help to add more detail in the tooltip. Something like:

Collect metrics from Elastic Agents that use this configuration.
Collect logs from Elastic Agents that use this configuration.

Reply:
thanks Dede! I'll add these